homeassistant/components/homekit_controller/climate.py

```diff
@@ -62,8 +64,11 @@ def update_characteristics(self, characteristics):
                 self._features |= SUPPORT_OPERATION_MODE
                 self._current_mode = MODE_HOMEKIT_TO_HASS.get(
                     characteristic['value'])
+
+                valid_values = characteristic.get(
+                    'valid-values', DEFAULT_VALID_MODES)
```

Comment:
This doesn't look right. We're mixing homekit modes with hass modes. DEFAULT_VALID_MODES is a list of hass modes.